client/test/e2e/enterprises/enterprises.spec.js

```diff
@@ -0,0 +1,83 @@
+/*global describe, it, element, by, beforeEach, inject, browser */
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+
+var FormUtils = require('../shared/FormUtils');
+
+describe('The Enterprises Module', function () {
+
+  // shared methods
+  var path = '#/enterprises';
+  var ENTERPRISES = {
+    name : 'Interchurch Medical Assistance',
+    abbr : 'IMA', 
+    email : 'ima@imaworldhealth.com', 
+    po_box : 'POBOX USA 1', 
+    phone : '01500', 
+    currency_id : 1
+  };
+
+  function update(n) {
+    return element(by.repeater("e in EnterpriseCtrl.enterprises | orderBy:'name'").row(n))
+      .$$('a')
+      .click();
+  }
+
+  // navigate to the enterprise module before each test
+  beforeEach(function () {
+    browser.get(path);
+  });
+
+  it('Successfully creates a new Enterprise', function () {
+    FormUtils.buttons.create();
+
+    FormUtils.input('EnterpriseCtrl.enterprise.name', ENTERPRISES.name);
+    FormUtils.input('EnterpriseCtrl.enterprise.abbr', ENTERPRISES.abbr);
+    FormUtils.input('EnterpriseCtrl.enterprise.email', ENTERPRISES.email);
+    FormUtils.input('EnterpriseCtrl.enterprise.po_box', ENTERPRISES.po_box);
+    FormUtils.input('EnterpriseCtrl.enterprise.phone', ENTERPRISES.phone);
+
+    // select a random, non-disabled option
+    FormUtils.select('EnterpriseCtrl.enterprise.location_id')
+      .enabled()
+      .first()
+      .click();
+
+    FormUtils.radio('EnterpriseCtrl.enterprise.currency_id', ENTERPRISES.currency_id);  
+    // submit the page to the server
+    FormUtils.buttons.submit();
```

Comment:
What should be returned after this?  Normally, in a test, you have an `expect()` condition.  What do you expect to happen?  How can you test that it worked?